charts/datadog/values.yaml

```diff
@@ -396,8 +396,11 @@ clusterAgent:
   # clusterAgent.token -- Cluster Agent token is a preshared key between node agents and cluster agent (autogenerated if empty, needs to be at least 32 characters a-zA-z)
   token: ""
 
-  # clusterAgent.tokenExistingSecret -- Existing secret name to use for Cluster Agent token
-  tokenExistingSecret: ""
+  # clusterAgent.tokenExistingSecret -- Existing secret name to use for Cluster Agent token, useful if you have a Cluster Agent deployed outside of this chart release
+  tokenExistingSecret:  # <EXISTING_DCA_SECRET_NAME>
+
+  # clusterAgent.existingService -- Existing service name to use for reaching the Cluster Agent, useful if you have a Cluster Agent deployed outside of this chart release
+  existingService:  # <EXISTING_DCA_SERVICE_NAME>
```

Comment:
maybe we can create a specific section:
```yaml
config:
  existingClusterAgent:
     join: false
     serviceName: "cluster-agent"
     tokenSecretName: ""
```

wdyt?

Reply:
yeah for sure, but be aware that i haven't tested it for every single feature